_posts/2015-11-02-lauer-astmh-presentation.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "ASTMH 2015 Presentation: Real-time prediction of dengue fever in Thailand"
+layout: post
+author: Steve
+comments: True
+---
+
+Last week, I had the honor of presenting at the 64th Annual Meeting of the [American Society of Tropical Medicine & Hygiene (ASTMH)](https://www.astmh.org/Home.htm) in the well-attended [Dengue: Epidemiology session](http://www.abstractsonline.com/Plan/ViewSession.aspx?sKey=b42307ef-cc96-4877-9cfe-40cb3e3ffb51&mKey=%7bAB652FDF-0111-45C7-A5E5-0BA9D4AF5E12%7d). This presentation covers our work with the Thai Ministry of Public Health and Johns Hopkins University in building an infrastructure for making real-time dengue hemorrhagic fever case predictions and evaluating the performance of our predictions thus far.
+
+You can find the slides for the presentation [here](https://speakerdeck.com/salauer/real-time-prediction-of-dengue-fever-in-thailand). After the jump, I'll provide a slide-by-slide summary.
+
+<!--more-->
+
+#### Slides 1-3: Introduction, funding, and project team members
+
+#### Slide 4: Statement of purpose and background
+
+The goal of this project is to predict Thailand dengue hemorrhagic fever (DHF) outbreaks in real time. Since 1980, there have been between 17,000 and 175,000 DHF cases reported each year, with about 50,000 on average. All four dengue serotypes co-circulate in Thailand, which creates complex transmission dynamics. Furthermore, there are 77 provinces, which vary in nearly every possible dimension from geography and climate to economy and demographics. Altogether, this makes it very difficult to have one unifying model for the entire country
+
+#### Slide 5: Presentation overview
+
+#### Slide 6: Data pipeline
+
+Every two weeks, the Thai Ministry of Public Health (MoPH) sends us a cumulative list of all the cases reported for the year-to-date. We integrate this into our PostgreSQL database and aggregate the data into analysis datasets. We use these datasets to make predictions, which we present to the Thai MoPH as a pdf and an interactive web app.
+
+#### Slide 7: Interactive web app
+
+With our interactive web app, the Thai MoPH can view the observed cases and our predictions for the rest of the year as a plot or on a heat map of Thailand. The app can be viewed in either English or Thai. In this screenshot, you can observe that 2013 was a large epidemic season, while 2014 was rather low. Thus far, we have already observed more cases for 2015 than for 2014, though we believe it is unlikely that there will be as many cases this year as in 2013.
```

Comment:
not sure about that last part - Derek is convinced we are going to see a big year, and this downplays that threat.